tests/integ/scala/test_datatype_suite.py

```diff
@@ -472,6 +474,118 @@ def test_structured_dtypes_iceberg(
         structured_type_session.sql(f"drop table if exists {table_name}")
 
 
+@pytest.mark.skip(
+    reason="SNOW-1748140: Need to handle structured types in datatype_mapper"
```

Comment:
how do you manual run this? did you manual did the fix for this one? who is hanlding this ticket, and when this will be done?

Reply:
I worked with @sfc-gh-jrose to add the fix at appropriate place and ran the test manually. Jamison will handle the ticket since he worked on supporting structured type support in snowpark.